<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectionDialog</class>
 <widget class="QDialog" name="ConnectionDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Connection setup</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>271</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Connection setup</string>
   </property>
   <widget class="QPushButton" name="detectLANButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Detect LAN ...</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clearHistoryButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>90</y>
      <width>121</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear history</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>17</y>
      <width>251</width>
      <height>61</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="nameLabel">
       <property name="text">
        <string>Name:</string>
       </property>
       <property name="buddy">
        <cstring>nameLineEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameLineEdit">
       <property name="enabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="hostLabel">
       <property name="text">
        <string>Host:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="hostComboBox">
       <property name="editable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="connectButton">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Connect</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="reconnectionCheckBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>143</y>
     <width>95</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Reconnection</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>ConnectionDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>188</x>
     <y>273</y>
    </hint>
    <hint type="destinationlabel">
     <x>241</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
